db/schema.rb

```diff
@@ -99,11 +99,27 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.uuid "content_change_id", null: false
+    t.bigint "or_joined_subscriber_list_id"
     t.index ["content_change_id", "subscriber_list_id"], name: "index_matched_content_changes_content_change_subscriber_list", unique: true
     t.index ["content_change_id"], name: "index_matched_content_changes_on_content_change_id"
     t.index ["subscriber_list_id"], name: "index_matched_content_changes_on_subscriber_list_id"
   end
 
+  create_table "or_joined_subscriber_list_subscriber_lists", force: :cascade do |t|
+    t.integer "subscriber_list_id"
+    t.integer "or_joined_subscriber_list_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["id", "or_joined_subscriber_list_id"], name: "idx_or_joined_sub_list_sub_list_on_or_joined_sub_list_id"
+    t.index ["id", "subscriber_list_id"], name: "idx_or_joined_sub_list_sub_list_on_sub_list_id"
```

Comment:
These names read as having one too many `sub_list`s - is this something we have control over or is it enforced by Rails?

Reply:
fortunately, this table wasnt actually necessary so it's gone